web/thesauruses/c/17/operators.json

```diff
@@ -0,0 +1,202 @@
+	},
+  "concepts": {
+    "addition": {
+      "code": "a+b",
+      "name": "Addition operator"
+    },
+    "addition_assignment": {
+      "code": "a+=b",
+      "name": "Addition and assignment operator"
+    },
+    "subtraction": {
+      "code": "a-b",
+      "name": "Subtraction operator"
+    },
+    "subtraction_assignment": {
+      "code": "a-=b",
+      "name": "Subtraction and assignment operator"
+    },
+    "multiplication": {
+      "code": "a*b",
+      "name": "Multiplication operator"
+    },
+    "multiplication_assignment": {
+      "code": "a*=b",
+      "name": "Multiplication and assignment operator"
+    },
+    "division": {
+      "code": "a/b",
+      "name": "Division operator"
+    },
+    "division_assignment": {
+      "code": "a/=b",
+      "name": "Division and assignment operator"
+    },
+    "integer_division": {
+      "not-implemented": "true",
+      "name": "Integer Division operator"
+    },
+    "integer_division_assignment": {
+      "not-implemented": "true",
+      "name": "Integer Division and assignment operator"
+    },
+    "modulus": {
+      "code": "a%b",
+      "name": "Modulus (remainder) operator"
+    },
+    "modulus_assignment": {
+      "code": "a%=b",
+      "name": "Modulus and assignment operator"
+    },
+    "unary_plus": {
+      "code" : "+a",
+      "name" : "Unary plus operator"
+  },
+    "unary_minus": {
+      "code" : "-a",
+      "name" : "Unary minus operator"
+  },
+    "increment": {
+      "code": "++a",
+      "name": "Increment (add 1) operator"
+    },
+    "decrement": {
+      "code": "--a",
+      "name": "Decrement (subtract 1) operator"
+    },
+    "exponential": {
+      "not-implemented": "true",
+      "name": "Exponential operator"
+    },
+    "factorial": {
+      "not-implemented": "true",
+      "name": "Factorial operator"
+    },
+    "absolute_value": {
+      "not-implemented": "true",
+      "name": "Absolute value operator"
+    },
+    "percentage": {
+      "not-implemented": "true",
+      "name": "Percentage operator"
+    },
+    "equal_to": {
+      "code": "a==b",
+      "name": "Equality operator"
+    },
+    "not_equal_to": {
+      "code": "a!=b",
+      "name": "Not equal to operator"
+    },
+    "less_than": {
+      "code": "a<b",
+      "name": "Less than operator"
+    },
+    "less_than_or_equal_to": {
+      "code": "a<=b",
+      "name": "Less than or equal to operator"
+    },
+    "greater_than": {
+      "code": "a>b",
+      "name": "Greater than operator"
+    },
+    "greater_than_or_equal_to": {
+      "code": "a>=b",
+      "name": "Greater than or equal to operator"
+    },
+    "null_coalescing": {
+      "not-implemented": "true",
+      "name": "Null coalescing operator"
+    },
+    "is": {
+      "not-implemented": "true",
+      "name": "Is operator"
+    },
+    "is_not": {
+      "not-implemented": "true",
+      "name": "Is not operator"
+    },
+    "logical_and": {
+      "code" : "if((a==1)&&(b==1))",
```

Comment:
This should JUST be the operator.  `a && b`.  You do this with operators, you should do it with these too. No `if`s.